modules/page/html/render/channel.js

```diff
@@ -36,7 +36,11 @@ exports.create = function (api) {
         ])
       ]),
       api.message.html.compose({
-        meta: {type: 'post', channel},
+        meta: Struct({
```

Comment:
You can avoid turning these into structs by using `mutant/resolve`:

```js
var resolve = require('mutant/resolve')
var draftLocation = resolve(meta.location) 
// now it doesn't matter if meta.location is an observable or just an ordinary key.
```

Reply:
Nice thanks.